frontend/app/_components/HomePageComponents/CoursesGrid.jsx

```diff
@@ -0,0 +1,82 @@
+import React from "react";
+import Image from "next/image";
+
+const CourseGrid = () => {
+  const courses = [
+    {
+      image: "/js.png",
+      name: "Introduction to Javascript",
+      description: "Learn Javascript FullStack",
+      instructor: "HackYourFuture",
+      price: "20.00",
+      sellingprice: "10.00",
+    },
+    {
```

Comment:
Same data is hardcoded in every page? Should consider not doing this, it's not good code practise